db/views/latest_blood_pressures_per_patient_per_months_v06.sql

```diff
@@ -0,0 +1,20 @@
+SELECT DISTINCT ON (patient_id, year, month)
+    blood_pressures.id AS bp_id,
+    blood_pressures.patient_id AS patient_id,
+    patients.registration_facility_id AS registration_facility_id,
+    patients.assigned_facility_id AS assigned_facility_id,
+    patients.status as patient_status,
+    blood_pressures.facility_id AS bp_facility_id,
+    blood_pressures.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE 'UTC' AS bp_recorded_at,
+    patients.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE 'UTC' AS patient_recorded_at,
+    blood_pressures.systolic AS systolic,
+    blood_pressures.diastolic AS diastolic,
+    blood_pressures.deleted_at AT TIME ZONE 'UTC' AT TIME ZONE 'UTC' AS deleted_at,
+    medical_histories.hypertension as medical_history_hypertension,
+    cast(EXTRACT(MONTH FROM blood_pressures.recorded_at AT TIME ZONE 'UTC' AT TIME ZONE (SELECT current_setting('TIMEZONE'))) as text) AS month,
```

Comment:
This is the same as before, where we grab the `recorded_at` as UTC, convert to the reporting time zone (assuming its set in the PG session), and then grab the month / year / quarter from it.